libraries/SITL/SIM_JSBSim.cpp

```diff
@@ -29,6 +29,8 @@
 
 extern const AP_HAL::HAL& hal;
 
+#define SIM_RATE_HZ 1000
```

Comment:
Instead of this, the `rate_hz` variable defined in SIM_Aircraft can be used
https://github.com/ArduPilot/ardupilot/blob/master/libraries/SITL/SIM_Aircraft.h#L169